### ISSUE TITLE
Before you start page

### ISSUE THREAD

**Issue by govuk-design-system** [alphagov/govuk-design-system-backlog#112]
## What
An information page that immediately follows a start page. It helps users understand what they need to do before they start using the service.

## Why

Qualitative research for Carer's Allowance found that GOV.UK content alone wasn't making people aware that applying for Carer's Allowance may (in certain circumstances) affect the benefits of the person they care for. Data analysis found that people were dropping out of the service when this was made clear to them later in their journey.

## Anything else
- [Before you start: DWP design patterns](https://dwp-patterns.herokuapp.com/beforeyoustart)

**Comment by timpaul** [comment]
Some questions:
- how would we decide which information goes on a start page and which on this page?
- does it matter that these pages would not be indexed or findable via Google?
- are these needs better met by asking direct questions, rather than hoping they read content?

**Comment by jfranciswebdesign** [comment]
Has anyone tested this yet? 

We're finding (via eye-tracking, and post-test questions) that users are completely ignoring the 'before you start' content on the [Start page](https://design-system.service.gov.uk/patterns/start-pages/) of our service because they're focussed on the big green button, so ignoring anything _under_ it.

Putting the content on a separate, dedicated 'Before you start' page seems like a way to get the user to read the content, but having this '_Before_ you Start' content _after_ the Start page seems illogical. I suppose a change to the page's title would solve this though - 'Before you continue'?!

**Comment by titlescreen** [comment]
@jfranciswebdesign Did you ever consider having the before you start above the big green button?

We were pleasantly surprised to see people skimming through our before you apply section. I'm not sure if they were originally honing in on the line `the application should take around 15 to 45 minutes to complete` then reading the before you apply section.

![image](https://user-images.githubusercontent.com/13239597/74345104-7371c580-4da5-11ea-81ea-ad06fc40e9a8.png)

**Comment by jfranciswebdesign** [comment]
@titlescreen Yes, and it tested much better! The issue is that the [GDS start page](https://design-system.service.gov.uk/patterns/start-pages/) has that content _after_ the Start button - "This is a set pattern, so you will not be able to customise it".

I recall, maybe 2 or 3 years ago, I had a pretty heated conversation with the GOV.UK person responsible for creating the Start page for our service. That person was quite insistent (to say the least!) that "GOV decides how start pages look, not you" this was after I'd explained that we wanted it different based on user research!

**Comment by dalegmoore** [comment]
How does one get access to some eye-tracking software?

**Comment by matthew20177** [comment]
Ripping it
